src/test/Withdrawals.t.sol

```diff
@@ -289,14 +208,12 @@ contract WithdrawalTests is EigenLayerTestHelper {
         cheats.warp(uint32(block.timestamp) + 2 days);
         cheats.roll(uint32(block.number) + 2);
 
-        uint256 prevElement = uint256(uint160(address(generalServiceManager2)));
-        generalReg1.propagateStakeUpdate(operator, uint32(block.number), prevElement);
+        // uint256 prevElement = uint256(uint160(address(generalServiceManager2)));
```

Comment:
Can remove these comments and probably also the warp/rolls

Reply:
agreed, but am feeling OK leaving this for later / whenever we rework this test. I think there's a lot more in this file that can get deleted.